src/status_im/ui/components/icons/vector_icons.cljs

```diff
@@ -2,121 +2,198 @@
-            :icons/chats               (components.svg/slurp-svg "./resources/icons/chats.svg")
-            :icons/hamburger           (components.svg/slurp-svg "./resources/icons/hamburger.svg")
-            :icons/hidden              (components.svg/slurp-svg "./resources/icons/hidden.svg")
-            :icons/in-contacts         (components.svg/slurp-svg "./resources/icons/in_contacts.svg")
-            :icons/lock                (components.svg/slurp-svg "./resources/icons/lock.svg")
-            :icons/mic                 (components.svg/slurp-svg "./resources/icons/mic.svg")
-            :icons/ok                  (components.svg/slurp-svg "./resources/icons/ok.svg")
-            :icons/public              (components.svg/slurp-svg "./resources/icons/public.svg")
-            :icons/public-chat         (components.svg/slurp-svg "./resources/icons/public_chat.svg")
-            :icons/qr                  (components.svg/slurp-svg "./resources/icons/QR.svg")
-            :icons/input-commands      (components.svg/slurp-svg "./resources/icons/input_commands.svg")
-            :icons/input-send          (components.svg/slurp-svg "./resources/icons/input_send.svg")
-            :icons/back                (components.svg/slurp-svg "./resources/icons/back.svg")
-            :icons/forward             (components.svg/slurp-svg "./resources/icons/forward.svg")
-            :icons/dropdown-up         (components.svg/slurp-svg "./resources/icons/dropdown_up.svg")
-            :icons/up                  (components.svg/slurp-svg "./resources/icons/up.svg")
-            :icons/down                (components.svg/slurp-svg "./resources/icons/down.svg")
-            :icons/grab                (components.svg/slurp-svg "./resources/icons/grab.svg")
-            :icons/share               (components.svg/slurp-svg "./resources/icons/share.svg")
-            :icons/tooltip-triangle    (components.svg/slurp-svg "./resources/icons/tooltip-triangle.svg")
-            :icons/open                (components.svg/slurp-svg "./resources/icons/open.svg")
-            :icons/network             (components.svg/slurp-svg "./resources/icons/network.svg")
-            :icons/wnode               (components.svg/slurp-svg "./resources/icons/wnode.svg")
-            :icons/refresh             (components.svg/slurp-svg "./resources/icons/refresh.svg")
-            :icons/newchat             (components.svg/slurp-svg "./resources/icons/newchat.svg")
-            :icons/logo                (components.svg/slurp-svg "./resources/icons/logo.svg")
-            :icons/camera              (components.svg/slurp-svg "./resources/icons/camera.svg")
-            :icons/check               (components.svg/slurp-svg "./resources/icons/check.svg")
-            :icons/dots                (components.svg/slurp-svg "./resources/icons/dots.svg")
-            :icons/warning             (components.svg/slurp-svg "./resources/icons/warning.svg")})
+(if platform/desktop?
```

Comment:
wouldn't it work to do something like
`(def slurp-svg
   (if plateform/desktop? js/require components.svg/slurp-svg))`
and then just use slurp-svg once for all instead of repeating everything like this ?
(not sure because js/require might be special ?)

Reply:
@yenda, I tried that approach and it didn't work. Not sure why, I thought that problem is in slurp-svg which is macros.
Also, that will result in fails in the production build. When javascript bundle prepared for the app, all require calls processed and required modules added to the bundle. When hiding js/require behind other `def`, processing doesn't happen and modules code not included in result bundle.

Reply:
Thanks for the info, have you tried doing that in the slurp-svg macro then ?

Reply:
Do you mean to put `js/require` inside slurp-svg in case of desktop build? No didn't try.
Good idea, thank you, I'll check.

@yenda, do you mind if we leave this check as an improvement after this pr merged? We have a problem merging it because while full building/testing happens, new commits arise in `develop` and Jenkins fails to make a rebase on them. That leads to a necessity to recreate PR and do a full cycle again.
Just now we agreed with @chu and @annadanchenko, that this PR will be tested immediately after it was built by Jenkins and we want to merge it quickly until new conflicts arise.

Yor other notes already addressed.